ui/src/store/auth.test.ts

```diff
@@ -141,4 +141,29 @@ describe('Store functions', () => {
 
     done();
   });
+
+  it('can update refresh and access token', (done) => {
+    const store = AuthStore.create({ accessTokenInfo: {}, refreshTokenInfo: {} }, { api });
+
+    const code = {
+      code: 'foo'
+    };
+    store.authenticate(code);
+    store.updateAccessToken();
+    store.updateRefreshToken();
+    expect(store.isLoading).toBe(true);
+    when(
+      () => !store.isLoading,
+      () => {
+        setTimeout(() => {
+          expect(store.isAuthenticated).toBe(true);
+
+          expect(store.accessTokenInfo.expiresAt).toBe(1611888148);
+          expect(store.refreshTokenInfo.expiresAt).toBe(1611888989);
```

Comment:
can we add a comment over here so that it's clear how many min or hrs or days it represents?